packages/hardhat-core/src/internal/hardhat-network/provider/modules/web3.ts

```diff
@@ -32,9 +34,7 @@ export class Web3Module {
   }
 
   private async _clientVersionAction(): Promise<string> {
-    const hardhatPackage = await getPackageJson();
-    const ethereumjsVMPackage = require("@nomicfoundation/ethereumjs-vm/package.json");
-    return `HardhatNetwork/${hardhatPackage.version}/@nomicfoundation/ethereumjs-vm/${ethereumjsVMPackage.version}`;
+    return this._node.getClientVersion();
```

Comment:
The value returned in the `clientVersion` field of the metadata is the same one that is returned by `web3_clientVersion`. Is that ok?

Reply:
Yep